credsweeper/utils/util.py

```diff
@@ -160,12 +169,12 @@ def preprocess_file_diff(cls, changes: List[Dict]) -> List[DiffRowData]:
         for change in changes:
             if change.get("old") is None:
                 # indicates line was inserted
-                rows_data.append(DiffRowData(DiffRowType.ADDED, change.get("new"), change.get("line")))
```

Comment:
`.get` here allows to pass `None` in case of a missing value, preventing `KeyError` exception 
https://stackoverflow.com/a/11041421

I'm not sure that raising `KeyError` here is necessary or not
@Dmitriy-NK , what do you think?